tests/test_latentdistributiontest.py

```diff
@@ -18,25 +18,20 @@ def setUpClass(cls):
         cls.A2 = er_np(20, 0.3)
 
     def test_fit_p_ase_works(self):
-        npt = LatentDistributionTest()
+        npt = LatentDistributionTest(method="dcorr")
```

Comment:
let's have at least one test with directed input. At least one test where the input is an embedding, not a graph.